### DISCUSSION TITLE
Fonts loaded from google are now a legal problem

### DISCUSSION THREAD

**Discussion by Synchro** [laravel/nova-issues#3739] [General]
- Laravel Version: n/a
- Nova Version: 3.30
- PHP Version: n/a

### Description:

Something that has been raised before in #1933 and #1134 is Nova's use of google fonts. You made it clear back then that this was not going to change, however, something else has changed that makes quite a big difference: [using Google Fonts is now illegal in the EU](https://noyb.eu/en/austrian-dsb-eu-us-data-transfers-google-analytics-illegal). That article mainly discusses Google Analytics, but it applies equally to all assets loaded from the US including fonts and code CDNs (and yes, that is a big deal), along with [this verdict](https://www.dataprotectionauthority.be/iab-europe-held-responsible-for-a-mechanism-that-infringes-the-gdpr) that has Facebook threatening to withdraw from the EU altogether.

If you're not going to bundle fonts or allow self-hosting, would you at least consider proxying them (because it's client exposure that is the problem) or using automatic font replacement like [Spatie's google fonts](https://github.com/spatie/laravel-google-fonts) package provides?

**Comment by davidhemphill** [comment]
Users are free to remove these assets in v3.x from the layout files if their jurisdiction requires it. The next series of Nova will likely self-host the font files directly.

**Reply by Synchro** [reply]
Good to know, thanks

**Comment by baszach** [comment]
Hi, does Nova 4 self-host the font files? @davidhemphill

**Reply by crynobone** [reply]
Yes

![image](https://user-images.githubusercontent.com/172966/186886775-0a8e01c1-5a68-4340-aec4-35fa175ec625.png)

**Reply by baszach** [reply]
Thanks. I just wanted a verification to be on the safe side :)
